styles.css

```diff
@@ -0,0 +1,502 @@
+/* 7. Focusable visually-hidden elements */
+/* 8. Maintain layout while hiding elements */
+
+
+/* ------------------------------- 4. Media Defaults -------------------------------- */
+/* Resets voor media-elementen zoals afbeeldingen en video's */
+
+/* 9. Media defaults */
+
+/* ------------------------------- 5. Layout ---------------------------------------- */
+/* Regels of resets die te maken hebben met containergedrag en positioning */
+/* Zet hier ook Basisregels voor grids (indien van toepassing).  */
+
+
+/*.outer-wrapper {*/
+/*    display: flex;*/
+/*    flex-direction: column;*/
+/*    justify-content: center;*/
+/*    align-items: center;*/
+/*}*/
+
+/*.inner-content-container {*/
+/*    display: flex;*/
+/*    flex-direction: row;*/
+/*    justify-content: space-between;*/
+/*    align-items: center;*/
+/*    gap: 1rem;*/
+/*    padding: 0 1rem;*/
+/*}*/
+
+/*.flex-outer {*/
+/*    display: flex;*/
+/*    justify-content: center;*/
+/*    align-items: center;*/
+/*    gap: 2rem;*/
+/*}*/
+
+/*.flex-inner {*/
+/*    display: flex;*/
+/*    flex-flow: row wrap;*/
+/*    justify-content: space-between;*/
+/*    gap: 1rem;*/
+/*}*/
+
+/*.flex-column {*/
+/*    display: flex;*/
+/*    flex-direction: column;*/
+/*    gap: 1rem;*/
+/*}*/
+
+
+/* ------------------------------- 6. Areas ----------------------------------------- */
+/* Specifieke resetregels of basestijlen voor unieke secties zoals Header en Footer */
+/* Vul zelf verder aan met structuurelementen van de site  */
+
+image {
+    max-width: 20%;
+}
+
+/* --------------- 6.1 Header ------------------------- */
+
+.outer-header-wrapper {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+}
+
+.header-wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    padding: 1rem;
+    flex: 1 0 65vh;
+
+}
+
+.inner-header-container {
+    margin: 0 auto;
+    padding: 0 2rem; /* Ruimte aan de zijkanten voor kleinere schermen */
+
+}
+
+.inner-header-container {
+    display: flex;
+    flex-flow: row;
+    justify-content: end;
+    width: 100%;
+}
+
+.navi ul {
+    display: flex;
+    gap: 3rem;
+    list-style: none;
+}
+
+.navi ul li a {
+    color: var(--alt-color);
+    text-decoration: none;
+    font-weight: 300;
+}
+
+.title-container {
+    margin-top: 3rem;
+    text-align: center;
+    padding: 20px 0;
+}
+
+.title-container > h1 {
+    padding: 1rem;
+}
+
+.title-container > p {
+    font-size: 1.5rem;
+    padding: 0.5rem;
+}
+
+#header {
+    background: linear-gradient(172deg, var(--main-color) 0%, var(--main-color) 65%, var(--text2-color) 98%);
+    position: relative;
+    color: white;
+}
+
+#header::after {
+    content: '';
+    position: absolute;
+    bottom: -10px; /* Zorgt dat het pseudo-element over de main valt */
+    left: 0;
+    width: 100%;
+    height: 4rem;
+    background: var(--alt-color);
+    clip-path: polygon(0 0, 0 100%, 100% 100%, 100% 65%);
```

Comment:
Je hebt nu een clip-path gebruikt, maar een CSS-driehoekje (EdHub hfst. 9.1 is al voldoende (en minder "zwaar")